cmake/onnxruntime_unittests.cmake

```diff
@@ -952,14 +952,18 @@ if(WIN32 AND onnxruntime_ENABLE_INSTRUMENT)
   target_link_libraries(compare_two_sessions PRIVATE ${GETOPT_LIB_WIDE} tdh Advapi32)
 endif()
 
-onnxruntime_add_executable(onnxruntime_mlas_test ${TEST_SRC_DIR}/mlas/unittest.cpp)
+file(GLOB onnxruntime_mlas_test_src CONFIGURE_DEPENDS
+  "${TEST_SRC_DIR}/mlas/*.h"
+  "${TEST_SRC_DIR}/mlas/test_*.cpp"
```

Comment:
can the code be moved to a subdirectory under test\mlas\unittest, next to bench instead? you wouldn't need the test_* pattern anymore either, just *.cpp.

Reply:
no problem, will update it.